BangazonAPI/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,361 @@
+﻿using BangazonAPI.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomerController : ControllerBase
+
+        {
+            private readonly IConfiguration _config;
+            public CustomerController(IConfiguration config)
+            {
+                _config = config;
+            }
+
+            public SqlConnection Connection
+            {
+                get
+                {
+                    return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+                }
+            }
+
+            [HttpGet]
+            public async Task<IActionResult> Get(string include, string q)
+            {
+                using (SqlConnection conn = Connection)
+                {
+                    conn.Open();
+
+                    using (SqlCommand cmd = conn.CreateCommand())
+                    {
+
+                    //SqlCommands dependant on query strings
+
+                        string command = "";
+                        string customerColumns = @"SELECT Customer.Id, Customer.FirstName, Customer.LastName";
+
+                        string customerTable = " FROM Customer";
+
+                        string paymentColumns = ", PaymentType.Id AS PaymentTypeId, PaymentType.AcctNumber, PaymentType.Name AS PaymentName, PaymentType.CustomerId, PaymentType.IsActive";
+
+                        string paymentTable = " JOIN PaymentType ON Customer.Id = PaymentType.CustomerId";
+
+                        string productColumns = ", Product.Id as ProductId, Product.ProductTypeId, Product.CustomerId AS ProductCustomerId, Product.Price, Product.Title, Product.Description, Product.Quantity, Product.CustomerId";
+
+                        string productTable = " JOIN Product ON Product.CustomerId = Customer.Id";
+
+                        string searchTable = $" WHERE Customer.FirstName LIKE '%{q}%' OR Customer.LastName LIKE '%{q}%'";
+
+                        //Conditionals for query strings
+
+                        if (include == "paymentTypes")
```

Comment:
This all looks good, and I think go ahead and merge, but just FYI-- the ticket has different query strings than we've got here. (For example, the ticket says you should go to `?_include=payments` and you've got `?include=paymentTypes`. Not a huge deal, just something to make a note of and potentially refactor post MVP